tests/unit/dataactvalidator/test_fabs31_detached_award_financial_assistance_4.py

```diff
@@ -34,30 +35,33 @@ def test_pubished_date_success(database):
                                                           assistance_type="03", action_date="10/02/2010",
                                                           record_type=1, business_types="A")
     det_award_5 = DetachedAwardFinancialAssistanceFactory(awardee_or_recipient_uniqu="111111112",
+                                                          assistance_type="03", action_date="10/02/2010",
+                                                          record_type=1, business_types="A")
+    det_award_6 = DetachedAwardFinancialAssistanceFactory(awardee_or_recipient_uniqu="111111112",
```

Comment:
Should there be a success test for record_type 3 for completeness?

Reply:
Oops, I was supposed to switch that to a 3 and missed it. Good catch